redaxo/src/addons/mediapool/lib/media.php

```diff
@@ -87,6 +87,21 @@ public static function get($name)
         });
     }
 
+    /**
+     * @throws rex_sql_exception
+     * @return null|static
+     */
+    public static function fromId(int $mediaId): ?self
```

Comment:
zur info: jan und ich haben nochmal über den methodennamen gesprochen, weil `forId` komisch klingt.
und wir sind nun dafür, `fromX` für solche methoden zu nutzen. Das scheint mir das gängige Format für solche Methoden zu sein.

Mit dem Schema `forX` haben wir zwar drei Methoden (zwei Mal forKey, einmal forLogin), aber die kann man dann irgendwann mal (in R6 oder so) ins Schema fromX bringen. Erstmal würde ich die einfach so lassen, aber für neue  Methoden halt fromX verwenden.

Reply:
Bevor man jetzt ein 3. format anfängt dann doch lieber getById() wenn forId() nicht zusagt?

Warum klingt forId() komisch?

Reply:
Mir ist es im Prinzip egal ..  forId liest sich für mich nur komisch und ich kann damit nichts anfangen. Vielleicht auch nur weil es mir unbekannt ist und mir nicht gebräuchlich vorkommt. Für mich ist getById auch ok. Nur ungerne hie eine Grundsatzdiskussion starten.

Reply:
mir sind die beiden varianten die wir bereits haben recht. eine 3. würd ich nicht anfangen.. 

wg. mir kann man mit R6 dann das überdenken, aber jetzt noch ein weiteres Faß aufmachen und damit die inkonsistenz noch weiter vorran zu treiben find ich nicht sinnvoll.

Reply:
forId

Reply:
forId ;-)

Reply:
tss, frechheit ;)